src/egon/data/json_metadata/demand.egon_heat_idp_pool.json

```diff
@@ -24,8 +24,8 @@
   "temporal": {
     "referenceDate": "2022-09-14",
     "timeseries": {
-      "start": "0 h",
-      "end": "23 h",
+      "start": "2011-01-01 00:00",
+      "end": "2011-01-01 00:00",
```

Comment:
Shouldn't these two timestamps differ? Like go from "2011-01-01 00:00" to "2011-01-02 00:00" or something like that?

Reply:
Yes